images/router/nginx/README.md

```diff
@@ -0,0 +1,41 @@
+### 1. Build the image
+In nodes,build origin openshif/nginx-router image:
+ - git clone https://github.com/openshift/origin
+ - cd origin/images/router/nginx/
+ - docker build -t openshift/nginx-router .
+
+### 2. Run the nginx router
+In master, deploy and edit nginx-router so that working directory can be fixed
+ - oadm policy add-scc-to-user hostnetwork -z router
```

Comment:
Don't we need this below policy, similar to HAProxy?
`oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:default:router`